resolwe_bio/tests/processes/test_upload.py

```diff
@@ -722,3 +723,76 @@ def test_upload_bedpe(self):
         self.assertFile(bedpe, "bedpe", in_file)
         self.assertFields(bedpe, "species", species)
         self.assertFields(bedpe, "build", build)
+
+    @tag_process("upload-clinical-metadata")
+    def test_upload_metadata(self):
+        base = Path("test_metadata_upload")
```

Comment:
IMO `metadata_upload` would be sufficiently long name of the folder. No need for the `test_`, we already know it's that given the location of this subfolder.